### PR TITLE
F/ci undockerize

### LINKED ISSUE #182: Add install method to ci recipe
This should help as a boilerplate to update projects with the old circleci config by doing

`potassium install ci`

### LINKED ISSUE #364: Upgrade CircleCI build to a self contained version with cache
Currently we bundle apps with a ci build that uses complicated files (like bin/cibuild) and relies on an external Platanus-maintained image. Additionally, we don't leverage caché for the testing build, only for the linting one.

The proposed final ci config is available [here](https://github.com/rjherrera/ci-cache-presentation/blob/master/.circleci/config.yml).

### PR DESCRIPTION
**Pull request by rjherrera**
This PR closes #364 by updating the circle ci config to stop using cibuild/compose.ci/dockerfile.ci in favour of a self-contained plain config using the docker environment provided by circle ci. It uses caché for the testing job and reusable setup for both linting and testing jobs.

I also added the install method to the recipe in case its needed for older projects, so this closes #182, and updated the readme to suit the new ci flow.

### CODE REVIEW

**Review by jgmontoya — APPROVED** [review]
LGTM!

File: lib/potassium/recipes/ci.rb [hunk 1]
```diff
@@ -1,50 +1,20 @@
 class Recipes::Ci < Rails::AppBuilder
   def create
-    copy_file '../assets/Dockerfile.ci', 'Dockerfile.ci'
     template '../assets/.circleci/config.yml.erb', '.circleci/config.yml'
 
-    template '../assets/bin/cibuild.erb', 'bin/cibuild'
-    run "chmod a+x bin/cibuild"
-
-    copy_file '../assets/docker-compose.ci.yml', 'docker-compose.ci.yml'
```
Comment (difernandez) [inline lib/potassium/recipes/ci.rb:9]: Se podrían borrar el `Dockerfile.ci`, `bin/cibuild` y `docker-compose.ci.yml` de los assets

**Review by difernandez — APPROVED** [review]
Bueeen PR! Se ve bien, dejo aprobado con un comentario chico

File: lib/potassium/recipes/ci.rb (hunk 1 above)
Comment (rjherrera) [reply lib/potassium/recipes/ci.rb:9]: toda la reason

**Pull request merged**